playwright.config.ts

```diff
@@ -39,6 +42,7 @@ export default defineConfig({
     {
       name: "chromium",
       use: { ...devices["Desktop Chrome"] },
+      
     },
     {
       name: "firefox",
```

Comment:
remove other browsers for now and uncomment different form 
 {
      name: "chromium",
      use: { ...devices["Desktop Chrome"] },
    }
    
       one for the mobile app view.